libbeat/cmd/instance/beat_test.go

```diff
@@ -1,6 +1,6 @@
 // +build !integration
 
-package beat
+package instance
```

Comment:
I put it in libbeat/cmd, as the `instance` package is used by almost all commands defined in libbeat/cmd. This way all code required or requiring a beat instance is combined in libbeat/cmd.